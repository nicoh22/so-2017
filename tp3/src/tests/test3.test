load a1.txt a2.txt a3.txt

maximum

member casa1
member gato
member perro
member estoy

maximum

addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2
addAndInc casa2

maximum
