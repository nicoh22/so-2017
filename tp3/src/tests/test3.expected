El máximo es <auto,8>
El string <auto> está
El string <gato> está
El string <perro> está
El string <estoy> no está
El máximo es <auto,8>
El máximo es <bici,12>
